js/templates/home.html

```diff
@@ -49,9 +49,9 @@ <h1 class="fancy-heading fontSize20 vCentered js-discoverHeading"><%= polyglot.t
                 <span class="ion-person fontSize11 marginRight2 toggle-txt"></span>
                 <span class="toggle-txt"><%= polyglot.t('ShowOnlyFollowerListings') %></span>
               </a>
-              <% if(!localStorage.getItem('safeListingsWarningDissmissed')){ %>
```

Comment:
Why did you remove this? If you do that, the discover helper will appear when it shouldn't.